tests/openvino/tools/calibrate.py

```diff
@@ -51,9 +51,29 @@
 
 TModel = TypeVar("TModel")
 
+OVERRIDE_OPTIONS_ALGORITHMS = ["ActivationChannelAlignment", "FastBiasCorrection", "BiasCorrection"]
+
 MAP_POT_NNCF_ALGORITHMS = {
-    "DefaultQuantization": "quantize",
-    "AccuracyAwareQuantization": "quantize_with_accuracy_control",
+    "ActivationChannelAlignment": {
+        "method": "quantize",
+        "advanced_parameters": {"disable_channel_alignment": False},
```

Comment:
Could you please briefly explain why `"disable_bias_correction"` is not set to `True` here?